pkg/metrics/eventmetrics/eventmetrics.go

```diff
@@ -25,7 +25,7 @@ var (
 		Name:        "events_total",
 		Help:        "The total number of Tetragon events",
 		ConstLabels: nil,
-	}, []string{"type", "namespace", "workload", "pod", "binary"})
+	}, metrics.FilterMetricLabels("type", "namespace", "workload", "pod", "binary"))
```

Comment:
I had originally scoped metrics.FilterMetricLabels to only be used for filtering the globally configured labels.
The result was a statement that seems to fight the language:
`append([]string{"type"}, metrics.FilterMetricLabels(const.KnownMetricLabelFilters...)...`

We can't pass const.KnownMetricLabelFilters to the function because "type" is not a configuration label.
How do we want to handle this?

Reply:
I see. Can we use the global config here? Something like:
```
append([]string{"type"}, option.Config.MetricsLabelFilter...}
```
The config is set before the metrics are registered.

Reply:
We can't use the config here because the config is a map;
but we can use the consts.KnownMetricLabelFilters as you've suggested if you'd prefer that.
By using consts.KnownMetricLabelFilters, we do not have to assume labels in the config are in a specific order.

Reply:
Ok, let's use `consts.KnownMetricLabelFilters` then, we can live with an append.